packages/patternfly-4/react-charts/src/components/ChartArea/examples/ChartArea.md

```diff
@@ -242,6 +251,139 @@ class MultiColorChart extends React.Component {
 }
 ```
 
+## Multi-color chart with threshold indicators and responsive container
+
+```js
+import React from 'react';
+import {
+  Chart,
+  ChartArea,
+  ChartAxis,
+  ChartLegend,
+  ChartGroup,
+  ChartThreshold,
+  ChartThemeColor,
+  ChartThemeVariant
+} from '@patternfly/react-charts';
+
+class MultiColorChart extends React.Component {
+  constructor(props) {
+    super(props);
+    this.containerRef = React.createRef();
+    this.state = {
+      width: 0
+    };
+    this.handleResize = () => {
+      if (this.containerRef.current && this.containerRef.current.clientWidth) {
+        this.setState({ width: this.containerRef.current.clientWidth });
+      }
+    };
+  }
+
+  componentDidMount() {
+    this.handleResize();
+    window.addEventListener('resize', this.handleResize);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize);
+  }
+
+  render() {
+    const { width } = this.state;
+
+    return (
+      <div ref={this.containerRef}>
+        <div className="area-chart-threshold-bottom-responsive">
+          <Chart
+            ariaDesc="Average number of pets"
+            ariaTitle="Area chart example"
+            containerComponent={
+              <ChartVoronoiContainer
+                labels={({ datum }) => `${datum.name}: ${datum.y}`}
+                constrainToVisibleArea
+              />
+            }
+            legendPosition="bottom-left"
+            legendComponent={
+              <ChartLegend
+                data={[
+                  { name: 'Cats' },
+                  { name: 'Birds' },
+                  {
+                    name: 'Cats Threshold',
+                    symbol: { fill: ChartThemeColor.blue, type: 'threshold' }
+                  },
+                  {
+                    name: 'Birds Threshold',
+                    symbol: { fill: ChartThemeColor.orange, type: 'threshold' }
+                  }
+                ]}
+                itemsPerRow={2}
+              />
+            }
+            height={250}
+            padding={{
+              bottom: 100, // Adjusted to accomodate legend
```

Comment:
adding some additional height in new container class `area-chart-threshold-bottom-responsive` and inside the legend bottom padding to accommodate multi row legend